doc/extensions/tutorial.rst

```diff
@@ -65,18 +71,20 @@ contains the CKAN git repo. Let's use the parent directory instead:
 
 Now run the ``paster create`` command to create your extension::
 
-    paster --plugin=ckan create -t ckanext ckanext-iauthfunctions
-
-.. note::
+    cookiecutter https://github.com/<user>/<repo>.git
 
-   The last argument to the ``paster create`` command
-   (``ckanext-iauthfunctions`` in this example) is the name for your next
-   extension. CKAN extension names *have* to begin with ``ckanext-``.
-
-The command will ask you to answer a few questions. The answers you give will
+The command will present a few prompts. The information you give will
 end up in your extension's ``setup.py`` file (where you can edit them later if
 you want).
 
+.. note::
+
+   The first prompt is for is the name of your next
+   extension. CKAN extension names *have* to begin with ``ckanext-``. This
+   tutorial uses the project name ``ckanext-iauthfunctions``.
+   The prompts for ``project_shortname`` and ``plugin_class_name`` can be
+   ignored. They will be filled in automatically.
```

Comment:
The things mentioned in this note seem relatively clear to me during use. Does including the note just lengthen the docs unnecessarily?
```
$ cookiecutter https://github.com/f-osorio/cookiecutter-ckan-extension.git
project [Project's name (must begin with 'ckanext-')]: ckanext-hierarchy
keywords [A space seperated list of keywords]: tag1 tag2
description []: This is a great project!
author []: David
author_email []: david.read@hackneyworkshop.com
github_user_name [Github user or organization name]: davidread
project_shortname [hierarchy]:
plugin_class_name [HierarchyPlugin]:
```

Reply:
If it's clear then I agree that it's unnecessary. I wasn't sure how clear it is partly because it seems to me to be a little unintuitive to get a prompt for something that should be ignored. But there's no way around that.

`Cookiecutter` doesn't currently support having templates in subdirectories of repos. But there is a [PR for it and it's mostly through review](https://github.com/cookiecutter/cookiecutter/pull/1224).

Reply:
If cookiecutter doesn't support using a local path for a template then I'm fine with using another repo under the ckan org (kind of annoying for users to need internet access just to create an extension, but not worth maintaining a fork of cookiecutter)